modules/compute/gke-node-pool/main.tf

```diff
@@ -159,9 +161,9 @@ resource "google_container_node_pool" "node_pool" {
     }
 
     reservation_affinity {
-      consume_reservation_type = var.reservation_type
-      key                      = var.specific_reservation.key
-      values                   = var.specific_reservation.values
+      consume_reservation_type = var.reservation_affinity.consume_reservation_type
```

Comment:
did you also test the test case of passing `NO_RESERVATION` ? Also can we trigger the existing e2e ML GKE tests ?

Reply:
Yes.